content/learn/book/ecs/exclusive-world-access/_index.md

```diff
@@ -1,14 +1,138 @@
+- you're working with a `NonSend` resource that cannot be sent across threads
+- you're setting up tests to be run in a headless fashion
+- you're using `bevy_ecs` as a standalone crate
 
-TODO: discuss and demonstrate exclusive systems
+## Working with exclusive world access
 
-TOD: discuss and demonstrate `NonSend` resources
+### Basic usage
+
+Generally speaking, the [API](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html) of working with the `World` mirrors those elsewhere that you might be familiar with.
+
+Like with `Commands`, you can call `spawn`, `spawn_batch` and `despawn` to add and remove entities, adding components to them with `insert` and `insert_bundle`.
+Resources are simply accessed with [`get_resource::<R>`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_resource) and the mutable equivalent [`get_resource_mut::<R>`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_resource)
+
+Like with queries, you can call [`get::<C>`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get) and [`get_mut::<C>`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_mut) to access component data of a particular sort on a given entity.
+If you want access to *all* of the data on an entity, use [`get_entity`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_entity) and [`get_entity_mut](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.get_entity_mut) (or their faster but riskier siblings [entity](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.entity) and [`entity_mut`]), along with various [`EntityRef`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.EntityRef.html) methods.
+
+You can create new queries using [`query`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.query) and [`query_filtered`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.query_filtered), using the former when you only have one type parameter and the latter when you want to use the second filtering type parameter of standard queries as well.
+
+### Accessing multiple parts of the `World` simultaneously
+
+When working with non-trivial exclusive `World` logic, you're likely to run into cases where you need mutable access to more than one part of the `World` at once.
+This tends to make the compiler quite unhappy, but by carefully partitioning our data access we can ensure that our code doesn't violate Rust's memory safety rules.
+No `unsafe` needed!
+
+Right now, there are two main tools to do so:
+
+1. [`World::cell`](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.cell): Like the [concept of the same name from the Rust standard library](https://doc.rust-lang.org/std/cell/), this enables interior mutability by disabling Rust's compile-time checks for aliased mutability and replacing them run-time checks.
```

Comment:
I am not at all confident in this explanation; someone please chime in if you understand / can explain the mechanics better.